src/components/BackgroundGrid/index.module.scss

```diff
@@ -0,0 +1,59 @@
+@use "@scss/common" as *;
+
+.backgroundGrid {
+  position: absolute;
+  top: 0;
+  left: calc(var(--gutter-h));
+  width: calc(100% - var(--gutter-h) * 2 + 2px);
+  height: 100%;
+  z-index: -1;
+  border-left: 1px solid var(--theme-elevation-200);
+  border-right: 1px solid var(--theme-elevation-200);
+
+  /* background-size: 25% 100%;
+  background-repeat: repeat-x;
+
+  background: repeating-linear-gradient(
+    to right,
+    var(--theme-elevation-200),
+    var(--theme-elevation-200) 1px,
+    transparent 1px,
+    transparent 25%
+  ); */
+  /* border-right: 1px solid var(--theme-elevation-200); */
```

Comment:
@paulpopus Is this code still needed? If not, make sure to remove any commented out code - there are a few spots you have this happening.